maquetado/scss/detail.scss

```diff
@@ -1,31 +1,69 @@
 @import "colors";
 
 .book-title {
+  font-family: Roboto;
   font-size: 30px;
   font-weight: bold;
   margin-right: 10px;
+  line-height: 40px;
 }
 
 .book-detail-title {
   color: $black;
+  font-family: Roboto;
   font-size: 20px;
   font-weight: bold;
+  line-height: 25px;
   margin-right: 10px;
 }
 
 .book-description {
   color: $grey;
+  font-family: Roboto;
   font-size: 20px;
+  line-height: 25px;
 }
 
 .book-genre {
   color: $grey;
-  font-size: 25px;
   font-weight: bold;
+  font-size: 30px;
+  line-height: 40px;
+}
+
+.book-content-title {
+  display: flex;
+  flex-direction: column;
+  &::after {
+    background-color: $green;
+    content: "";
```

Comment:
Change double quotes with single quotes.

Reply:
EsLint fix it. when it press ctrl + s. put double quotes